### PR TITLE
styles(stats-detectors): Update regression details from feedback

### PR DESCRIPTION
**Pull request by Zylphrex**
From some design feedback, this updates the regression issue details page and cleans up some of the UI.

### CODE REVIEW

File: static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx [hunk 1]
```diff
@@ -200,9 +199,6 @@ function PerformanceDurationRegressionIssueDetailsContent({
         <ErrorBoundary mini>
           <EventBreakpointChart event={event} />
         </ErrorBoundary>
-        <ErrorBoundary mini>
-          <EventSpanOpBreakdown event={event} />
-        </ErrorBoundary>
```
Comment (Zylphrex) [inline static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx:205]: To be added back in the sidebar

File: static/app/components/events/eventStatisticalDetector/lineChart.tsx [hunk 2]
```diff
@@ -39,19 +40,18 @@ function LineChart({
...
-      ...percentileSeries,
-      ...intervalSeries.filter(s => !s.markArea), // get rid of the shading
-    ];
+    return [...percentileSeries, ...intervalSeries];
   }, [percentileSeries, evidenceData, theme]);
 
-  const rightSeries = useMemo(() => [throughputSeries], [throughputSeries]);
+  const rightSeries = useMemo(() => {
+    return [throughputSeries];
+  }, [throughputSeries]);
```
Comment (Zylphrex) [inline static/app/components/events/eventStatisticalDetector/lineChart.tsx:54]: Need to figure out how to draw the breakpoint here

Comment (narsaynorath) [reply static/app/components/events/eventStatisticalDetector/lineChart.tsx:54]: There's a util called `getIntervalLine` a few lines up, if you make another series and apply this `markLine` property:
```
  periodDividingLine.markLine = {
    data: [
      {
        xAxis: divider,
      },
    ],
    label: {show: false},
    lineStyle: {
      color: theme.textColor,
      type: 'solid',
      width: 2,
    },
    symbol: ['none', 'none'],
    tooltip: {
      show: false,
    },
    silent: true,
  };
```

this should get you that breakpoint iirc

**Review by narsaynorath — APPROVED** [review]
Awesome

**Pull request merged**